database/migrations/2016_06_01_000004_create_oauth_clients_table.php

```diff
@@ -18,6 +18,7 @@ public function up()
             $table->unsignedBigInteger('user_id')->nullable()->index();
             $table->string('name');
             $table->string('secret', 100)->nullable();
+            $table->string('provider')->nullable();
```

Comment:
I guess this would be a breaking change for anyone who already ran the migrations? Sounds like we need to move the pr to the master branch?

Reply:
Yeah, most likely. Sorry, I didn't realize it was against the tagged release.

Reply:
No sorry needed. Feel free to send it to master :)